obsah/__init__.py

```diff
@@ -359,6 +382,21 @@ def generate_ansible_args(inventory_path, args, obsah_arguments):
     return ansible_args
 
 
+def reset_args(application_config, metadata: dict, args: argparse.Namespace):
+    try:
+        with open(application_config.persist_path()) as persist_file:
+            persist_params = yaml.safe_load(persist_file)
+        if persist_params:
+            for (reset_key, reset_values) in metadata['reset']:
+                print(f"reseting {reset_key} in {persist_params}")
+                if persist_params.get(reset_key) is not None and persist_params.get(reset_key) != getattr(args, reset_key):
+                    for arg in reset_values:
+                        delattr(args, arg)
```

Comment:
this resets them to (ansible) defaults, but we want to keep it if passed by the user on the CLI (and not from the preserved set)